pkg/resources/kafka/configmap.go

```diff
@@ -99,7 +99,10 @@ func (r *Reconciler) getConfigProperties(bConfig *v1beta1.BrokerConfig, id int32
 	if err := config.Set("metric.reporters", "com.linkedin.kafka.cruisecontrol.metricsreporter.CruiseControlMetricsReporter"); err != nil {
 		log.Error(err, "setting metric.reporters in broker configuration resulted an error")
 	}
-	bootstrapServers := getInternalListener(r.KafkaCluster.Spec.ListenersConfig.InternalListeners, id, r.KafkaCluster.Spec.GetKubernetesClusterDomain(), r.KafkaCluster.Namespace, r.KafkaCluster.Name, r.KafkaCluster.Spec.HeadlessServiceEnabled)
+	bootstrapServers, err := kafkautils.GetBootstrapServersService(r.KafkaCluster)
```

Comment:
Wouldn't be more efficient if `cruise.control.metrics.reporter.bootstrap.servers` is pointed at the local broker (the broker the reporter runs in) instead of `kafka-all-brokers`?

Also, should the `cruise.control.metrics.reporter` run in each broker or only in the controller broker?